build.gradle.kts

```diff
@@ -72,13 +63,19 @@ node {
     download = true
 }
 
-java {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+tasks.named<KotlinCompile>("compileKotlin") {
+    kotlinOptions.jvmTarget = "1.8"
```

Comment:
java 11 ?

Reply:
Endring av java-versjon bør tas som en egen oppgave. Endringen i build.gradle.kts her er bare for å få intelliJ til å bedre forstå hvilken jvmTarget den kan forvente.